pandas/core/frame.py

```diff
@@ -5245,6 +5245,100 @@ def _construct_result(self, result) -> "DataFrame":
         out.columns = self.columns
         return out
 
+    @Appender(
+        """
+Returns
+-------
+DataFrame
+    DataFrame that shows the differences stacked side by side.
+        The resulting index will be a MultiIndex with 'self' and 'other'
+        stacked alternately at the inner level.
+
+See Also
+--------
+Series.differences: Show differences.
```

Comment:
```suggestion
Series.differences : Show differences.
```

Technically need a space between these; this *might* be the CI issue you are seeing